index.js

```diff
@@ -46,7 +47,7 @@ var gs = {
       stream.write({
         cwd: opt.cwd,
         base: basePath,
-        path: path.resolve(opt.cwd, filename),
+        path: filename,
```

Comment:
since the filename is (or should be) absolute now, I don't think it's needed. In case you're a visual person, like me, here is what we're getting back from glob now:

<img width="775" alt="screen shot 2015-11-05 at 4 28 11 pm" src="https://cloud.githubusercontent.com/assets/383994/10982259/3e0f3f60-83da-11e5-802f-0502dd11e506.png">